src/promnesia/sources/pocket.py

```diff
@@ -7,22 +7,43 @@
 def index() -> Results:
     from . import hpi
     from my.pocket import articles
+
     # TODO use docstring from my. module? E.g. describing which pocket format is expected
 
     for a in articles():
-        loc = Loc.make(title='pocket', href=a.pocket_link)
+        title = a.json.get('resolved_title', 'pocket')
```

Comment:
Yeah thanks, I guess this makes sense. 
Although sometimes `resolved_title` is `""` in my Pocket export. There is also `given_title` which is not empty for such entries, so maybe worth trying to use that too while we're at it?